src/atomate2/qchem/jobs/base.py

```diff
@@ -0,0 +1,158 @@
+"""Definition of a base QChem Maker"""
+
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+from pathlib import Path
+from typing import Callable
+
+from jobflow import Maker, Response, job
+from monty.serialization import dumpfn
+from monty.shutil import gzip_dir
+from pymatgen.core.structure import Molecule
+from pymatgen.io.qchem.inputs import QCInput
+
+from atomate2.qchem.files import copy_qchem_outputs, write_qchem_input_set
+from atomate2.qchem.run import run_qchem, should_stop_children
+from atomate2.qchem.schemas.task import TaskDocument
```

Comment:
Is this already in emmet? if so you can now use that one provided any atomate2 changes are moved across.